test/test_helper.rb

```diff
@@ -41,4 +41,4 @@ def admin_login
     return admin
   end
 
-end
\ No newline at end of file
+end
```

Comment:
Thank you for adding the carriage return to the end of the file. It will prevent false positives in the future.